toparea-text-widget/README.md

```diff
@@ -0,0 +1,76 @@
+# jupyterlab_toparea_text
```

Comment:
Would you mind explain what the extension code is doing in the README as we do for the other examples?

Reply:
Ahh sorry, I updated the description in the root README but not here. Will do!